.github/workflows/main.yml

```
@@ -30,7 +30,8 @@ jobs:
         run: |
           coverage run -m pytest tests
           coverage xml
-
+      - name : Run Bandit Security Scan
```

actionlint (1.7.7)

step must run script with "run" section or run action with "uses" section

(syntax-check)